File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("check-e-mail-or-username-for-a-data-breach_sdk")

describe("CheckEMailOrUsernameForADataBreachSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: go/test/exists_test.go
package sdktest

import (
	"testing"

	sdk "github.com/voxgig-sdk/check-e-mail-or-username-for-a-data-breach-sdk/go"
)

func TestExists(t *testing.T) {
	t.Run("test-mode", func(t *testing.T) {
		testsdk := sdk.TestSDK(nil, nil)
		if testsdk == nil {
			t.Fatal("expected non-nil SDK")
		}
	})
}
